common-lib/src/test/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverterTest.scala

```diff
@@ -83,19 +83,19 @@ class ImageMetadataConverterTest extends FunSpec with Matchers {
   it("should populate the dateTaken field of ImageMetadata from EXIF Date/Time Original Composite (Mon Jun 18 01:23:45 BST 2018)") {
     val fileMetadata = FileMetadata(iptc = Map(), exif = Map(), exifSub = Map("Date/Time Original Composite" -> "Mon Jun 18 01:23:45 BST 2018"), xmp = Map())
     val imageMetadata = ImageMetadataConverter.fromFileMetadata(fileMetadata)
-    imageMetadata.dateTaken should be (Some(parseDate("2018-06-18T01:23:45")))
+    imageMetadata.dateTaken should be (Some(parseDate("2018-06-18T00:23:45Z")))
```

Comment:
This test and the two below were always incorrect because the `BST` timezone difference wasn't being accounted for as the UTC version is in fact an hour previous.